src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/ResourceGroupTests.ps1

```diff
@@ -88,7 +88,8 @@ function Test-CreatesAndRemoveResourceGroupViaPiping
     New-AzureRmResourceGroup -Name $rgname1 -Location $location
     New-AzureRmResourceGroup -Name $rgname2 -Location $location
 
-    Get-AzureRmResourceGroup | where {$_.ResourceGroupName -eq $rgname1 -or $_.ResourceGroupName -eq $rgname2} | Remove-AzureRmResourceGroup -Force
+    Get-AzureRmResourceGroup | where {$_.ResourceGroupName -eq $rgname1 -or $_.ResourceGroupName -eq $rgname2} | Remove-AzureRmResourceGroup -Force -AsJob
+	Get-Job | Wait-Job
```

Comment:
Instead, assign the output to a variable and pass the variable in the the job cmdlets as a positional parameter.

i.e.

```powershell
$job = (Get-AzureRmResourceGroup ... | Remove-AzureRmResourceGroup -AsJob...)
Wait-Job $job
...
```

Reply:
Apply this to all the additional tests

Reply:
done